e2e/cypress/integration/markdown/markdown_image_spec.js

```diff
@@ -24,35 +22,83 @@ describe('Markdown', () => {
 
     it('with in-line images 1', () => {
         // #  Post markdown message
-        cy.postMessageFromFile('markdown/markdown_inline_images_1.md').wait(TIMEOUTS.SMALL);
+        cy.postMessageFromFile('markdown/markdown_inline_images_1.md');
 
         // * Verify that HTML Content is correct.
-        // Note we use the Gigantic timeout to ensure that the large images will load
+        // Note we check width and height to verify that img element is actually loaded
         cy.getLastPostId().then((postId) => {
-            cy.get(`#postMessageText_${postId}`).find('p').
-                should('have.text', 'Mattermost/platform build status:  ');
+            cy.get(`#postMessageText_${postId}`).find('div').
+                should('have.text', 'Mattermost/platform build status:  ').
+                and('have.class', 'style--none');
 
             cy.get(`#postMessageText_${postId}`).find('img').
                 should('have.class', 'markdown-inline-img').
                 and('have.attr', 'alt', 'Build Status').
-                and('have.attr', 'src', `${baseUrl}/api/v4/image?url=https%3A%2F%2Ftravis-ci.org%2Fmattermost%2Fplatform.svg%3Fbranch%3Dmaster`);
+                and('have.attr', 'src', `${baseUrl}/api/v4/image?url=https%3A%2F%2Ftravis-ci.org%2Fmattermost%2Fplatform.svg%3Fbranch%3Dmaster`).
+                and('have.css', 'height', '22px').
```

Comment:
Could you please check this again?  It's failing on me where the actual value is `'23px'`.